deployment/src/test/resources/application-splunk-logging-named-handler.properties

```diff
@@ -6,5 +6,11 @@ quarkus.log.handler.splunk."MONITORING".url=http://localhost:8088
 quarkus.log.handler.splunk."MONITORING".token=12345678-0000-0000-0000-1234567890AB
 quarkus.log.handler.splunk."MONITORING".level=INFO
 quarkus.log.handler.splunk."MONITORING".metadata-index=mystatsindex
+# Avoid batching and send events immediately, to make unit tests more synchronous
```

Comment:
Test LoggingSplunkNamedHandlerConfigTest is ko without this
It was working previously thanks to default batch-size-bytes being 10bytes, lower than the logged message `"{\"key\":\"value\"}"`